<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearPublicacion</class>
 <widget class="QDialog" name="crearPublicacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="agregarPublicacionBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Publicacion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contenidoEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>hola</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fechaPublicacionEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10/10/2023</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="horaPublicacionEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10:10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONTENIDO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FECHA DD/MM/YYYY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HORA HH:MM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seleccionarImagenBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccion Imagen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imagenPathLine">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imagenLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cerrarBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
